### PR TITLE
DOCS-10793 template creation troubleshooting

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-10793 template creation troubleshooting

### CODE REVIEW

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/troubleshooting.adoc [hunk 1]
```diff
@@ -6,6 +6,12 @@ endif::[]
 
 If you encounter errors while you are using Anypoint API Community Manager, refer to these explanations and instructions for recovery.
 
+== Error When Creating a New Community from a Template
+
+If creating a new community from a template shows the error "An unexpected error occurred. Try again or contact your Salesforce administrator for help," compile all Apex classes by opening *Setup* > *Custom Code* > *Apex Classes* and clicking *Compile all classes*. In a few minutes the page displays the message *Compilation Complete: Compilation Successful*.
```
Comment (kbostrom) [inline modules/ROOT/pages/troubleshooting.adoc]:
```suggestion
If creating a new community from a template shows the error `An unexpected error occurred. Try again or contact your Salesforce administrator for help`, compile all Apex classes by opening *Setup* > *Custom Code* > *Apex Classes* and clicking *Compile all classes*. In a few minutes the page displays the message `Compilation Complete: Compilation Successful`.
```
Bold is for UI labels ("permanent" UI element names), whereas mono is for individual user inputs and system outputs.

File: modules/ROOT/pages/troubleshooting.adoc [hunk 2]
```diff
@@ -6,6 +6,12 @@ endif::[]
...
+
+Find more information about compiling Apex classes in the Salesforce help article https://help.salesforce.com/articleView?id=sf.code_manage_packages.htm&type=5[Manage Apex Classes, role=external, window=blank].
```
Comment (kbostrom) [inline modules/ROOT/pages/troubleshooting.adoc:13]: Does this link create a maintenance issue?

**Review by kbostrom — DISMISSED** [review]
CE

File: modules/ROOT/pages/troubleshooting.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/troubleshooting.adoc]: Thank you! Fixed.

File: modules/ROOT/pages/troubleshooting.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/troubleshooting.adoc:13]: Salesforce help pages follow a policy of providing redirects if they change, so I think we can count on Salesforce to keep the link valid.

**Review by kbostrom — APPROVED** [review]
RA

**Pull request merged**